packages/language/src/document.js

```diff
@@ -567,16 +626,20 @@ Document.prototype.detectEntities = function(options, callback) {
 /**
  * Detect the sentiment of the text in this document.
  *
- * @resource [documents.analyzeSentiment API Documentation]{@link https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment}
+ * @resource [documents.analyzeSentiment API Documentation]{@link https://cloud.google.com/natural-language/reference/rest/v1/documents/analyzeSentiment}
  *
  * @param {object=} options - Configuration object. See
- *     [documents.annotateText](https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment#request-body).
+ *     [documents.annotateText](https://cloud.google.com/natural-language/reference/rest/v1/documents/analyzeSentiment#request-body).
  * @param {boolean} options.verbose - Enable verbose mode for more detailed
  *     results. Default: `false`
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error occurred while making this request.
- * @param {number} callback.sentiment - A value in the range of `-100` to `100`.
- *     Large numbers represent more positive sentiments.
+ * @param {number} callback.sentiment.sentiment - A value in the range of `-100`
```

Comment:
Let's keep our default return the same; just return the score.

Default:
```js
sentiment = 33
```

Verbose:
```js
sentiment = {
  score: 33,
  magnitude: 4,
  sentences: [
    {
      text: {
        content: '...',
        beginOffset: '...'
      }
    }
  ],
  language: 'en'
}
```